features/web-animations.yml

```diff
@@ -1,6 +1,70 @@
 name: Web animations
-description: The web animations API allows you to animate and synchronize the animations of DOM elements.
+description: The Web Animations API programmatically animates elements over time, and allows you to synchronize the animations of multiple elements in JavaScript.
```

Comment:
Maybe I'm off base here, but does this work as an entry point—it's the `compute_from` key, after all? Then we don't need the explicit note about scripting either.

```suggestion
description: The `animate()` method of `Element` objects programmatically animates elements over time and can synchronize the animations of multiple elements.
```

Reply:
Yes, I think this work.